packages/iris-grid/src/IrisGridModelUpdater.tsx

```diff
@@ -18,7 +19,6 @@ const COLUMN_BUFFER_PAGES = 1;
 
 interface IrisGridModelUpdaterProps {
   model: IrisGridModel;
-  modelColumns: readonly dh.Column[];
```

Comment:
Why are you removing this and using `model.columns` instead? Not sure anything in this file needs to be modified

Reply:
https://github.com/deephaven/web-client-ui/pull/2183#discussion_r1718831523

Reply:
Fair. Although the hook with `usePrevious` is not quite right. Previous won't be updated until after effects fire, so if a render is discarded due to something like a setState, then the change callback would fire again. At least I'm 95% sure that's how it works

Reply:
Checked w/ Akshat on changing `usePrevious` to use a ref instead of `useEffect` for the previous value and it caused a loading loop for the table, so left it as is.